app/src/main/java/com/codesoom/assignment/controllers/dtos/TaskResponseDto.java

```diff
@@ -0,0 +1,24 @@
+package com.codesoom.assignment.controllers.dtos;
+
+import com.codesoom.assignment.Task;
+
+public class TaskResponseDto {
+    private Long id;
+    private String title;
+
+    public TaskResponseDto() {
+    }
```

Comment:
```suggestion
    private final Long id;
    private final String title;
```

이런 형태가 되도록 개선해보면 좋은 경험이 될 것 같아요!

Reply:
@ssisksl77 DTO 내 필드에 대해 수정하는 메소드가 없으므로 DTO 객체를 불변으로 만들어 불면객체의 장점을 누리면 좋다고 생각합니다.
하지만 DTO 객체는 직렬화, 역직렬화 과정에서 자바빈 레거시에 따라 빈 기본생성자를 정의해야 합니다. 
결과적으로 final 키워드로 정의한 클래스 필드에 대해 초기화가 불가능하므로 DTO 객체는 불변객체로 정의할 수 없다고 판단했습니다.

Reply:
```java
public class TaskRequestDto {
    private String title;

    @ConstructorProperties({"title"})
    private TaskRequestDto(final String title) {
        this.title = title;
    }

    public String getTitle() {
        return title;
    }

    public Task toEntity() {
        return new Task(title);
    }
}
```

이 코드를 한번 참고해주시면 좋겠어요!

Reply:
멋진 노하우 공유해주셔서 감사합니다!!
Task 객체를 언박싱해서 내부의 멤버변수를 바로 가져오는 부분이 멋지네요
기본 생성자도 생략할 수 있다는 부분도 정말 매력적입니다 ㅎㅎ

Reply:
맥스웰님 작성해주신 코드 잘 보았습니다 ㅎㅎ
혹시 TaskRequestDTO와 TaskResponseDTO를 구분하신 이유에 대해 알 수 있을까요?
감사합니다~

Reply:
@LeeJongbokz 요구사항을 분석해보면 Request Body에 들어오는 값은 오직 title 뿐이죠. 하지만 Response Body로 내려줄 때는 title과 id를 함께 보내주어야 합니다.